src/main/java/com/singhinderjeet/countries/LanguageCode.java

```diff
@@ -24,122 +24,122 @@
+    pt_BR("Portuguese (BR)", CountryCode.BR, LanguageScript.Latn),
+    pt_PT("Portuguese (PT)", CountryCode.PT, LanguageScript.Latn),
+    ro_RO("Romanian (RO)", CountryCode.RO, LanguageScript.Latn),
+    ru_RU("Russian (RU)", CountryCode.RU, LanguageScript.Cyrl),
+    sr_BA("Serbian (BA)", CountryCode.BA, LanguageScript.Cyrl),
+    sr_ME("Serbian (ME)", CountryCode.ME, LanguageScript.Cyrl),
+    sr_RS("Serbian (RS)", CountryCode.RS, LanguageScript.Cyrl),
+    sr_Latn_BA("Serbian (BA)", CountryCode.BA, LanguageScript.Latn),
+    sr_Latn_ME("Serbian (ME)", CountryCode.ME, LanguageScript.Latn),
+    sr_Latn_RS("Serbian (RS)", CountryCode.RS, LanguageScript.Latn),
+    sk_SK("Slovak (SK)", CountryCode.SK, LanguageScript.Latn),
+    sl_SI("Slovenian (SI)", CountryCode.SI, LanguageScript.Latn),
+    es_AR("Spanish (AR)", CountryCode.AR, LanguageScript.Latn),
+    es_BO("Spanish (BO)", CountryCode.BO, LanguageScript.Latn),
+    es_CL("Spanish (CL)", CountryCode.CL, LanguageScript.Latn),
+    es_CO("Spanish (CO)", CountryCode.CO, LanguageScript.Latn),
+    es_CR("Spanish (CR)", CountryCode.CR, LanguageScript.Latn),
+    es_DO("Spanish (DO)", CountryCode.DO, LanguageScript.Latn),
+    es_EC("Spanish (EC)", CountryCode.EC, LanguageScript.Latn),
+    es_SV("Spanish (SV)", CountryCode.SV, LanguageScript.Latn),
+    es_GT("Spanish (GT)", CountryCode.GT, LanguageScript.Latn),
+    es_HN("Spanish (HN)", CountryCode.HN, LanguageScript.Latn),
+    es_MX("Spanish (MX)", CountryCode.MX, LanguageScript.Latn),
+    es_NI("Spanish (NL)", CountryCode.NL, LanguageScript.Latn),
+    es_PA("Spanish (PA)", CountryCode.PA, LanguageScript.Latn),
+    es_PY("Spanish (PY)", CountryCode.PY, LanguageScript.Latn),
+    es_PE("Spanish (PE)", CountryCode.PE, LanguageScript.Latn),
+    es_PR("Spanish (PR)", CountryCode.PR, LanguageScript.Latn),
+    es_ES("Spanish (ES)", CountryCode.ES, LanguageScript.Latn),
+    es_US("Spanish (US)", CountryCode.US, LanguageScript.Latn),
+    es_UY("Spanish (UY)", CountryCode.UY, LanguageScript.Latn),
+    es_VE("Spanish (VE)", CountryCode.VE, LanguageScript.Latn),
+    sv_SE("Swedish (SE)", CountryCode.SE, LanguageScript.Latn),
+    th_TH("Thai (TH)", CountryCode.TH, LanguageScript.Thai),
+    th_TH_u_ca_buddhist("Thai (TH)", CountryCode.TH, LanguageScript.Thai), // Implicitly specifies the Latin numbering system. Explicitly specifies the Script.Thai Buddhist calendar with java.util.Calendar.
+    th_TH_u_ca_buddhist_nu_thai("Thai (TH)", CountryCode.TH, LanguageScript.Thai), // Explicitly specifies the Script.Thai numbering system and the Script.Thai Buddhist calendar with java.util.Calendar.
+    th_TH_x_lvariant_TH("Thai (TH)", CountryCode.TH, LanguageScript.Thai), // Explicitly specifies the Thai numbering system and the Thai Buddhist calendar with java.util.Calendar. (For compatibility)
+    tr_TR("Turkish (TR)", CountryCode.TR, LanguageScript.Latn),
+    uk_UA("Ukrainian (UA)", CountryCode.UA, LanguageScript.Cyrl),
```

Comment:
If a language is unique, then just use the name. 
For example, uk_UA should only be "Ukrainian"

For languages which have multiple variants, add the country code:
es_PR ("Spanish (PR)")
es_US("Spanish (US)")